src/commonMain/kotlin/fr/acinq/lightning/bin/Main.kt

```diff
@@ -120,13 +120,14 @@ class Phoenixd : CliktCommand() {
             "5m" to 5_000_000.sat,
             "10m" to 10_000_000.sat,
         ).default(2_000_000.sat, "2m")
-        val maxAbsoluteFee by option("--max-absolute-fee", help = "Max absolute fee for on-chain operations. Includes mining fee and service fee for auto-liquidity.")
+        val maxAbsoluteFee by option("--max-absolute-fee", hidden = true).deprecated("--max-absolute-fee is deprecated, use --max-mining-fee instead", error = true)
+        val maxMiningFee by option("--max-mining-fee", help = "Max mining fee for on-chain operations, in satoshis")
             .int().convert { it.sat }
-            .restrictTo(5_000.sat..100_000.sat)
-            .defaultLazy("2% of auto-liquidity amount") {
-                autoLiquidity * 2 / 100
+            .restrictTo(5_000.sat..200_000.sat)
```

Comment:
I'm increasing the max allowed value. In case of a fee spike, some users may want to accept higher mining fees instead of rejecting payments, there is no reason to be excessively conservative.